cli/flox/src/commands/services/mod.rs

```diff
@@ -316,8 +320,7 @@ pub async fn start_services_with_new_process_compose(
         use_fallback_interpreter: false,
         // FIXME: This should match the current activation.
         mode: Some(ActivateMode::Dev),
-        // FIXME: This should match the current activation.
-        generation: None,
+        generation,
```

Comment:
suggestion nonblocking: ephemeral activation to start services at a specific generation seems about as brittle as it gets. Might be worth a test?